mozjs/js/moz.configure

```diff
@@ -271,13 +271,13 @@ def profiling(value):
 
 add_old_configure_assignment('MOZ_PROFILING', profiling)
 
-@depends(profiling, target)
-def imply_vtune(value, target):
+@depends(profiling, target, js_enable_uwp)
+def imply_vtune(value, target, uwp):
     ok_cpu    = target.cpu in ['x86', 'x86_64']
     ok_kernel = target.kernel == 'WINNT' or \
                 (target.kernel == 'Linux' and target.os == 'GNU')
 
-    if value and ok_cpu and ok_kernel:
+    if value and ok_cpu and ok_kernel and not uwp:
```

Comment:
Why is this patch needed if we explicitly disable `vtune` when invoking `configure`? I don't get it.

Reply:
Without being patched, configute complained that --enable-profiling always enabled vtune and you could not disable it.

Reply:
Oh so `--disable-vtune` has less weight than this implied setting? Good to know.